_sub/compute/eks-nodegroup-managed/dependencies.tf

```diff
@@ -39,3 +39,15 @@ locals {
 data "aws_eks_cluster" "this" {
   name = var.cluster_name
 }
+
+# --------------------------------------------------
+# Inactivity based clean up for sandboxes
+# --------------------------------------------------
+
+locals {
```

Comment:
This conditional statement doesn't make sense to me. What it basically says is:

IF var.enable_inactivity_cleanup is **null** THEN use **null**. Otherwise use var.eks_is_sandbox

Reply:
What is you set the variable enable_inactivity_cleanup to true, and then change the logic below to:

```
locals {
  enable_inactivity_cleanup = (
    var.enable_inactivity_cleanup && var.eks_is_sandbox ? true: false
  )
}
```